model/src/main/proto/spotlight.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// Superclass for all the SpotlightViewStates
+message Spotlight {
+  oneof feature {
+    // feature for onboarding screen next button
```

Comment:
This should be a complete sentence. Furthermore, I think you could make it a bit clearer. Consider:

```suggestion
    // Corresponds to the onboarding screen's next button.
```

Ditto for the others below.

Reply:
done